HAL_extern.h

```diff
@@ -153,7 +152,6 @@ void    initConfKey(void) {
 
 #ifndef USE_OWN_ISR_PCINT2_vect
 	ISR (PCINT2_vect) {
-		pcInt[2].prev = pcInt[2].cur;
```

Comment:
Es bringt nichts, im Interrupt den Prev-Status zu setzen.
Wenn die Taste prellt, dann ist ab dem 2. Interrupt prev und cur gleich.
Deshalb im Interrupt NUR den neuen, aktuellen Status setzen und in prev 
den vorigen, entprellten Status - gegen den dann der neue Status 
verglichen werden kann.

Am 03.05.2016 um 23:52 schrieb Dirk:
>
> In HAL_extern.h 
> <https://github.com/trilu2000/NewAskSin/pull/16#discussion_r61962777>:
>
> > @@ -153,7 +152,6 @@ void    initConfKey(void) {
> >
> >  #ifndef USE_OWN_ISR_PCINT2_vect
> >  	ISR (PCINT2_vect) {
> > -		pcInt[2].prev = pcInt[2].cur;
>
> Warum fällt pcInt[(0-2)].prev weg?
> Das fällt dann spätestens in chkPCINT() auseinander. Denn dort wird 
> das noch verwendet.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/trilu2000/NewAskSin/pull/16/files/9ca902b175462b05c199500fbb2955e9055a978c#r61962777>
>